samples/bluetooth/mesh/light/README.rst

```diff
@@ -145,6 +183,12 @@ Configure the Generic OnOff Server model on each element on the **Mesh Light** n
 
 Make sure to complete the configuration on each of the elements on the node to enable controlling each of the remaining three LEDs.
 
+Running DFU
+===========
+
+After the sample is built with the :file:`overlay-dfu.conf` and programmed to your development kit, support for FOTA upgrades is enabled.
+See FOTA in Bluetooth mesh (link) for instructions on how to perform FOTA upgrade and initiate the DFU process.
```

Comment:
I assume this `(link)` is gonna be made into an actual link before this is merged?

Reply:
Ah of course Mia explained this while I was reviewing. Then I guess it's fine

Reply:
Mia will fix it 👍